package.json

```diff
@@ -27,12 +27,14 @@
   "homepage": "https://github.com/primer/primitives#readme",
   "scripts": {
     "build": "ts-node ./script/build.ts && tsc",
-    "build:tokens": "node -e \"require('./build')._init()\"",
+    "build:tokens": "ts-node -e \"require('./build')._init()\"",
+    "build:buildJs": "tsc --project newToken.tsconfig.json",
```

Comment:
I think we could run tsc and ts-node sequentially to avoid having two commands.

Reply:
well, we don't need `tsc` when we build the tokens. And once we get `ts-jest` we only need it for `primer/brand` this is why I think we could keep it separate.

Reply:
cool cool 👍 ... how about `build:self-serve` then? `build:buildJs` could be too ambiguous for new folks browsing the repo.